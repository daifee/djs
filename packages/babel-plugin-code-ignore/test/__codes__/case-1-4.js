// @code-ignore-next-statement kugou kuwo
// 
console.log('aaaa')


