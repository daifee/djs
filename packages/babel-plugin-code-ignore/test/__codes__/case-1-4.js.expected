"use strict";

// @code-ignore-next-statement kugou kuwo
// 
console.log('aaaa');